release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
++
+If a CSV defines a conversion webhook but does not include an API service or an admission webhook, the CSV cycles through the "Pending", "ReadyToInstall", and "Installing" phases indefinitely. This bug fix updates OLM to use the existing conversion webhook to retrieve the value of the CA and correctly calculate the deployment hash. As a result, OLM can now install CSVs that define a conversion webhook without an API service or admission webhook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885398[*BZ#1885398*])
+
+* In the `opm` command, the `semver-skippatch` mode previously allowed only bundles with later patch versions as valid replacements, ignoring any pre-release versions. Bundles with the same patch versions but later pre-release versions were not accepted as replacements. This bug fix updates the `opm` command to base the `semver-skippatch` check on the semantic versioning as a whole instead of just the patch version. As a result, later pre-release versions are now valid for the `semver-skippatch` mode. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1889721[*BZ#1889721*])
+
+* Previously, the Marketplace Operator did not clean stale services during a cluster upgrade, and Operator Lifecycle Manager (OLM) accepted the stale service without validating the service. This caused the stale service to direct traffic to a catalog source pod that contained outdated content. This bug fix updates OLM to add spec hash information to the service and check to ensure the service has the correct spec by comparing the hash information. OLM then deletes and recreates the service if it is stale. As a result, the service spec now directs traffic to the correct catalog source pod. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1891995[*BZ#1891995*])
+
+* After mirroring an Operator to a disconnected registry, the Operator install could fail due to a missing related bundle image. This issue was due to the bundle image not being present in the `index.db` database. This bug fix updates the `opm` command to ensure the bundle image is present in the `related_images` table of the database. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1895367[*BZ#1895367*])
+
+* Previously, Operator authors could create cluster service versions (CSVs) that define webhooks with container ports set outside of the 1 to 65535 range. This prevented the `ValidatingWebhookConfiguration` or `MutatingWebhookConfiguration` objects from being created because of validation failure. CSVs could be created that never successfully install. The custom resource definition (CRD) validation for CSVs now includes the proper minimum and maximum values for the `webhookDescription` `ContainerPort` field. This now defaults to `443` if the container port is not defined. CSVs with invalid container ports now fail validation before the CSV is created. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1891898[*BZ#1891898*])
```

Comment:
I think the tense is wrong here and "that never successfully install" should be "that never successfully installed"